Makefile

```diff
@@ -227,3 +223,22 @@ create-metrics-svc:
 delete-metrics-svc:
 	kubectl delete -f deploy/example/metrics/csi-azuredisk-controller-svc.yaml --ignore-not-found
 
+.PHONY: setup-external-e2e
+setup-external-e2e:
+	curl -sL https://storage.googleapis.com/kubernetes-release/release/v1.19.0/kubernetes-test-linux-amd64.tar.gz --output e2e-tests.tar.gz
+	tar -xvf e2e-tests.tar.gz
+	rm e2e-tests.tar.gz
+	mkdir /tmp/csi-azuredisk
+	cp ./kubernetes/test/bin/e2e.test /tmp/csi-azuredisk/e2e.test
+	rm -r kubernetes
+	cp ./deploy/example/storageclass-azuredisk-csi.yaml /tmp/csi-azuredisk/storageclass.yaml
+	cp ./test/external-e2e/manifest/testdriver.yaml /tmp/csi-azuredisk/testdriver.yaml
+	./deploy/install-driver.sh
+
+.PHONY: e2e-test
+e2e-test: setup-external-e2e
+	if [ ! -z "$(EXTERNAL_E2E_TEST)" ]; then \
+		bash ./test/external-e2e/run.sh;\
```

Comment:
`$(MAKE) setup-external-e2e` and then `bash ./test/external-e2e/run.sh`

Reply:
I think it's better put the logic of `setup-external-e2e` into script `./test/external-e2e/run.sh`

Reply:
done. updated.